### PR TITLE
Fix issues on automation feature

### PR DESCRIPTION
**Pull request by spdkimo**
1. Fix a temperature sensor issue that adding the device on automation keeps failing
2. Fix a humidity sensor issue that adding the device on automation keeps failing
3. On file save dialog, add .xml extension as default file name
4. Change the control from QComboBox to QSpinBox to change values on automation

### CODE REVIEW

**Review by junyong-sim — DISMISSED** [review]
I feel spinbox is more compotable than combobox when I've tested.

File: src/automation/automationmain.py [hunk 1]
```diff
@@ -50,6 +50,7 @@
 class automationWindow(QtWidgets.QMainWindow):
     dialog_closed = QtCore.pyqtSignal(str)
     send_used_list = QtCore.pyqtSignal(dict)
+    STR_TKN = ' : '
```
Comment (jinwon0325kim) [inline src/automation/automationmain.py:53]: Where use it?

Comment (spdkimo) [reply src/automation/automationmain.py]: It seems that the variable is unused anymore. Removed

**Review by jinwon0325kim — APPROVED** [review]
good job

**Pull request merged**